src/helpers/common.ts

```diff
@@ -1,5 +1,10 @@
 import { ReactNode } from "react";
 
+export type OmitR<
+  T extends Record<keyof unknown, unknown>,
+  K extends keyof T,
+> = Pick<T, Exclude<keyof T, K>>;
+
```

Comment:
Should we document what is the difference between the normal `Omit` type helper and this one?

Reply:
Good catch! This one is specifically for Records, so it's good to make it clear on JSDoc 🙂